packages/waku/src/router/create-pages.ts

```diff
@@ -237,6 +252,11 @@ export const createPages = <
     createLayout: CreateLayout;
     createRoot: CreateRoot;
     createApi: CreateApi;
+    /**
+     * Page Part pages are assumed to be dynamic in order to allow dynamic components.
```

Comment:
This sounds like a limitation.
Can't we make all page parts static and make the entire page static?

Reply:
Technically you can, but why would you?

If your goal is to make a static page, shouldn't you just use createPage?

createPagePart only seems useful to me when you're mixing dynamic and static which then requires the page to be dynamic, right?

Reply:
> shouldn't you just use createPage?

Technically, not always.

`getConfig` itself is dynamically configurable at build time.

```js
export const getConfig = async () => {
  return {
    render: Date.now() < SOME_POINT ? 'static' : 'dynamic',
  };
};
```

Reply:
I see what you mean. I can add logic for static page when all parts are static in that case. Thanks for the example.

Reply:
marking draft for now.

Reply:
@tylersayshi Can you fix the comment?